p/themes/Origine/origine.css

```diff
@@ -45,6 +45,10 @@ input, select, textarea {
 	vertical-align: middle;
 }
 
+select {
+	height: 37px;
```

Comment:
I am not a found of `px` unit for main dimensions of text-based elements (borders, minimal/maximal dimensions are fine with `px`).
Could you try to use another unit such as `em` ?

Reply:
I am with you, but the issue is the different box modells of `<input>` and `<select>`

Input: 1px border + 5px padding + 25px height + 5px padding + 1px border = 37px
Select: height = border + padding + X + padding + border

so the height of select needs a `px` height :/

Reply:
Have you tried playing around with box-sizing? (content-box for example)